modules/ROOT/pages/users.adoc

```diff
@@ -103,7 +103,11 @@ When a user logs into Anypoint Platform for the first time, they are automatical
 
 You cannot search for external users in other parts of Anypoint Platform because these users do not belong to your organization and do not have additional permissions. To grant users permission to perform tasks like deploying an application, you must invite the user to join your organization.
 
+== Managing Sensitive Account Information
 
+As an organization administrator, you can modify user some sensitive user information. For enhanced security, Access Management might ask you to re-authenticate before you can modify an organization user's email address.
```

Comment:
```suggestion
As an organization administrator, you can modify user some sensitive user information. For enhanced security, Access Management might ask you to re-authenticate before you can modify an organization user's email address. Because of the re-authentication requirement, you cannot use API calls to change user email addresses.
```